sortinghat/core/models.py

```diff
@@ -233,8 +233,12 @@ def __str__(self):
         return '%s - %s' % (self.individual.mk, self.organization.name)
 
 
-class MatchingBlacklist(EntityBase):
-    excluded = CharField(max_length=MAX_SIZE_CHAR_FIELD, primary_key=True)
+class RecommenderExclusionList(EntityBase):
+    excluded = CharField(max_length=MAX_SIZE_CHAR_INDEX)
 
     class Meta:
-        db_table = 'matching_blacklist'
+        db_table = 'recommender_exclusion_list'
```

Comment:
This should be kept with this name because that's the table where these objects are stored.